app/actions/local/channel.ts

```diff
@@ -189,7 +189,16 @@ export const markChannelAsViewed = async (serverUrl: string, channelId: string,
     }
 };
 
-export const markChannelAsUnread = async (serverUrl: string, channelId: string, messageCount: number, mentionsCount: number, manuallyUnread: boolean, lastViewed: number, prepareRecordsOnly = false) => {
+export const markChannelAsUnread = async (
+    serverUrl: string,
+    channelId: string,
+    messageCount: number,
+    mentionsCount: number,
+    manuallyUnread: boolean,
```

Comment:
I don't see a case where `manuallyUnread` is not going to be true, at least from the existing code... so this should also be inferred